test/expectations/expectations_test_runner.rb

```diff
@@ -110,27 +110,33 @@ def ruby_requirement_magic_comment_version(fixture_path)
   private
 
   def test_addon(addon_creation_method, source:)
-    stub_no_typechecker
     message_queue = Thread::Queue.new
+    stub_no_typechecker
```

Comment:
Is this a cosmetic change or there's some order-dependent side-effects?

Reply:
Just cosmetic. I actually think I was trying to remove `message_queue`, but then I realized we need it to load the addons.